corehq/apps/accounting/async_handlers.py

```diff
@@ -234,6 +234,19 @@ def domain_response(self):
         return [(d, d) for d in domain_names]
 
 
+class Select2CustomerInvoiceTriggerHandler(BaseSelect2AsyncHandler):
+    slug = 'select2_billing'
+    allowed_actions = [
+        'customer_account',
+    ]
+
+    @property
+    def customer_account_response(self):
+        customer_accounts = BillingAccount.objects.filter(is_customer_billing_account=True)
+        customer_account_names = [account.name for account in customer_accounts]
```

Comment:
here you can just do `BillingAccount.objects.filter(is_customer_billing_account=True).values_list('name', flat=True)` instead of pulling all the values from the db.